python/src/scipp/plot/plot_1d.py

```diff
@@ -177,23 +178,21 @@ def __init__(self,
 
         return
 
+    def get_finite_y(self, arr):
+        if self.logy:
+            with np.errstate(divide="ignore", invalid="ignore"):
+                arr = np.log10(arr, out=arr)
+        subset = np.where(np.isfinite(arr))
```

Comment:
Could also just call `np.nonzero` directly.

Reply:
Did you want to see this change before I merge or is it ok to merge? (I have lots of np.where in other places)

Reply:
No, fine to keep consistent.